docs/topics/about-tools.adoc

```diff
@@ -1,14 +1,22 @@
 // Module included in the following assemblies:
 //
-// * docs/cli-guide/master.adoc
+// * docs/getting-started-guide/master.adoc
 
 [id="about-tools_{context}"]
 = About the tools
 
-The {ProductName} ({ProductShortName}) provides several tools to assist you in the various stages of your migration and modernization efforts. Review the details of each tool to determine which is right for your project.
+You can use the following {ProductName} ({ProductShortName}) tools to help you migrate and modernize projects:
 
-* CLI
-* Web console
-* {PluginName} for Eclipse and Red Hat CodeReady Studio
-* {ProductShortName} extension for Visual Studio Code
-* {MavenName}
+* link:{ProductDocUserGuideURL}[CLI]: Assess and prioritize migration and modernization efforts for applications without the overhead of a graphical user interface by using the {CLIName} tool.
```

Comment:
I see a potential problem with all these links. The user will assume that the link points to the tool itself, and not to the guide. You might want to remove the link at the beginning and add a sentence after each description like "See the X Guide [link] for more information."

I don't like the phrase "without the overhead ..." because it sounds slangy and marketer-like. But if that's what the team wants, leave it. Personally, I would look for other reasons to use the CLI (like automation or speed) rather than saying that it avoids the hassle of using the UI.